### ISSUE TITLE
ng serve documentation (specifically --public-host/live-reload-client)

### ISSUE THREAD

**Issue by dtabaka** [angular/angular-cli#8986]
### Versions
```
Node 6.11.2, Npm 5.6.0, Angular CLI 1.5.0, Angular 5, Asp.NET Core 2.0, Visual Studio 2017
```
### Repro steps

I'm integrating the Angular CLI with Visual Studio 2017 & an ASP.NET Core 2.0 application where I successfully have the live reloading working when I make a change to my Angular components.  I have the following script tags in my _layout.cshtml during development.

  <script type="text/javascript" src="http://localhost:4200/inline.bundle.js"></script>
etc.

### Desired behavior
However, for this to work I stumbled upon a post that mentioned I had to configure my package.json "start" script as follows:

 "scripts": {
    "start": "ng serve  --live-reload-client http://localhost:4200/",

I would like:
1.) To better understand this and what it is doing behind the scenes.  From what I understand **ng serve** uses webpack dev server under the covers so I'm wondering how this configures it or what the equivalent setting in the webpack.config.json file would be.

2.) ng serve docs have this listed as **public-host** (aka **live-client-reload**) but not much more than that.  If there was more information it might help clear up 1.)

**Comment by Splaktar** [comment]
There are docs now in https://angular.io/cli/serve, but it's still not clear what `--public-host` really does and there are no examples of how/when this would be useful.

All it says is "Specify the URL that the browser client will use."

I came to the docs trying to fix an issue with live reload on Chrome OS. Not mentioning live reload and public host are related cost me a good bit of time and frustration.

@jbogarthyde @jenniferfell this docs issue is closer to being addressed now. I think a little bit of refinement to the existing `ng serve` docs should solve this.

**Comment by Splaktar** [comment]
This appears to be resolved in `7.3.0` via https://github.com/angular/angular-cli/commit/4729397a680010ceb79b8cdfea96295a74e16715.

**Comment by Splaktar** [comment]
But it appears that previous PR only changed one entry for this, but there are multiple. https://github.com/angular/angular-cli/pull/14538 is working to address this for the dev-server builder.

The [documentation for this on AIO](https://next.angular.io/cli/serve) has not been updated or improved, so this issue is not yet resolved.

**Comment by Splaktar** [comment]
It looks like the [`dev-server` builder](https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/build_angular/src/dev-server/schema.json) schema is the right place. That change in #14538 should cause the AIO docs to update properly and fix this issue.

**Comment by jbogarthyde** [comment]
Fixed by  PR #14538
